backend/app/sql/create_assignment_partition_sql_udf.sql

```diff
@@ -0,0 +1,51 @@
+CREATE FUNCTION create_assignment_partition_sql(document_id text)
+       RETURNS text LANGUAGE plpgsql AS $$
+BEGIN
+  RETURN format('CREATE TABLE assignments_%s PARTITION OF assignments
```

Comment:
I think we'll need to double quote assignments_%s since uuids have dashes in them